backend/src/dfini_hack_team4.did

```diff
@@ -0,0 +1,59 @@
+type Amount = nat64;
+type AnswerId = nat64;
+type Solution = blob;
+type TaskId = nat64;
+type TaskPayload = blob;
+type Timestamp = nat64;
+
+type TaskType = variant {
+  translate_text;
+  edit_image;
+};
+
+type TaskStatus = variant {
+  open;
+  closed;
+};
+
+type Choice = variant {
+  yes;
+  no;
+};
+
+type Vote = record {
+  voter: principal;
+  choice: Choice;
+};
+
+type Answer = record {
```

Comment:
again wondering if we should either stick with "answer" or "solution", but no big deal.